test/functional/relative-time/format-relative-time.js

```diff
@@ -0,0 +1,97 @@
+define( [
+	"globalize",
+	"json!cldr-data/main/en/dateFields.json",
+	"json!cldr-data/main/de/dateFields.json",
+	"json!cldr-data/main/en/numbers.json",
+	"json!cldr-data/main/de/numbers.json",
+	"json!cldr-data/supplemental/likelySubtags.json",
+	"json!cldr-data/supplemental/numberingSystems.json",
+	"json!cldr-data/supplemental/plurals.json",
+	"../../util",
+
+	"globalize/number",
+	"globalize/relative-time"
+], function( Globalize, enDateFields, deDateFields, enNumbers, deNumbers,
+	likelySubtags, numberingSystems, plurals, util ) {
+
+var en, de;
```

Comment:
If you don't mind, let's sort this alphabetically: `var de, en`. Such criterion helps on further changes.